ci/unittests_linux.sh

```diff
@@ -24,16 +26,31 @@ TESTLOGS=$TRAVIS_BUILD_DIR/logs/test_logs;
 grep "Global test environment tear-down" $TESTLOGS
 retVal=$?
 count=0
-while [ "$retVal" -ne 0 ] &&  [ "$count" -ne 180 ]; do
+corefile=1
+while [ "$retVal" -ne 0 ] &&  [ "$count" -ne 180 ]i && [ "$corefile" -eq 1 ] ; do
 	sleep 60;
 	grep "Global test environment tear-down" $TESTLOGS
 	retVal=$?
+        ls /tmp/pxscenecrash
+        corefile=$? 
 	count=$((count+60))
 	echo "unittests running for $count seconds"
 done
 
-echo "kill -9 `ps -ef | grep pxscene2dtests |grep -v grep|grep -v pxscene2dtests.sh|awk '{print $2}'`"
-kill -9 `ps -ef | grep pxscene2dtests |grep -v grep|grep -v pxscene2dtests.sh|awk '{print $2}'`
+#check for corefile presence
+processId=`ps -ef | grep pxscene2dtests |grep -v grep|grep -v pxscene2dtests.sh|awk '{print $2}'`
+ls -l /tmp/pxscenecrash
+retVal=$?
+if [ "$retVal" -eq 0 ]
+  then
+  $TRAVIS_BUILD_DIR/ci/check_dump_cores_linux.sh `pwd` "$TRAVIS_BUILD_DIR/tests/pxScene2d/pxscene2dtests" "$processId" "$TESTLOGS"
+  kill -9 $processId
```

Comment:
just want to make sure, if we send signal to a process which is already in signal handler, it is not creating any problem?.

Reply:
In this case, if the signal sent is a SIGKILL or SIGTERM, it will kill the process and we will not get the stack trace as this signal cannot be ignored and this script will exit gracefully. 
Whereas with other signals like SIGSEGV, the script is continued to get the stack trace.